packages/utilities/src/scroll.ts

```diff
@@ -156,25 +157,22 @@ export function findScrollableParent(startingElement: HTMLElement | null): HTMLE
 
   // First do a quick scan for the scrollable attribute.
   while (el && el !== document.body) {
+    let style = getComputedStyle(el);
+    if (style.position === 'fixed') {
+      return document.body;
+    }
     if (el.getAttribute(DATA_IS_SCROLLABLE_ATTRIBUTE) === 'true') {
       return el;
-    }
-    el = el.parentElement;
-  }
-
-  // If we haven't found it, the use the slower method: compute styles to evaluate if overflow is set.
-  el = startingElement;
-
-  while (el && el !== document.body) {
-    if (el.getAttribute(DATA_IS_SCROLLABLE_ATTRIBUTE) !== 'false') {
-      const computedStyles = getComputedStyle(el);
-      let overflowY = computedStyles ? computedStyles.getPropertyValue('overflow-y') : '';
-
-      if (overflowY && (overflowY === 'scroll' || overflowY === 'auto')) {
+    } else {
+      const overflowRegex = /(auto|scroll)/;
```

Comment:
I think there are behavior changes to this function which carry risk and are more important to verify.

However, I think if we pursue this approach, we should optimize this code for perf if it is called as frequently as I think (traversing upwards to find a scrollable parent).

Perhaps we can hoist this declaration.

>Regular expression literals provide compilation of the regular expression when the script is loaded. If the regular expression remains constant, using this can improve performance
>https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions